src/main/java/com/cachinginapp/enterprise/CachingInController.java

```diff
@@ -23,17 +32,24 @@ public class CachingInController {
      * Handle the root (/) endpoint and return a start page.
      * @return
      */
-
     @RequestMapping("/")
     public String index(Model model) {
         return "start";
     }
-
+    /**
+     * Map Page?
+     * TODO: Explain methods purpose, and any @params and @return
+     *
+     */
     @RequestMapping("/map")
```

Comment:
Add Javadoc and include purpose of method even if it hasn't been developed or utilized yet